posixlib/src/main/scala/scala/scalanative/posix/signal.scala

```diff
@@ -0,0 +1,407 @@
+package scala.scalanative
+package posix
+
+import scala.scalanative.unsafe._
+
+/**
+ * Some of the functionality described on this reference page extends the ISO C standard.
+ * Applications shall define the appropriate feature test macro (see XSH The Compilation Environment )
+ * to enable the visibility of these symbols in this header.
+ *
+ * Use clib macros and functions to get the full POSIX interface
+ *
+ * Note 1: The functionality described may be removed in a future version of this volume of POSIX.1-2017
+ * Note 2: The functionality is an option marked as XSI (X/Open System Interfaces)
+ */
+@extern
+object signal {
+  // define the following macros, which shall expand to constant expressions with distinct values
+  // that have a type compatible with the second argument to, and the return value of, the signal() function,
+  // and whose values shall compare unequal to the address of any declarable function.
+
+  // Note 1: Linux
+  //  @name("scalanative_sig_hold")
+  //  def SIG_HOLD: CFunctionPtr1[CInt, Unit] = extern
```

Comment:
Shouldn't this be uncommented? Same applies to code under another notes

Reply:
I cleaned these up and put a good comment at the top explaining. Basically, these are in the spec but don't work on either Linux or macOS. They are left in the code so a future developer will not go chasing these down. Comparing the spec to the code is all visual and then parts are in the portable `clib` so merging all these things together is really hard.